NinchatSDK/NINChatSession.m

```diff
@@ -297,7 +303,7 @@ -(void) continueSessionWithCredentials:(nonnull NINSessionCredentials*)credentia
 //        [weakSelf.sessionManager closeOldSession:credentials.sessionID];
```

Comment:
What is/was the purpose of this?

Reply:
This line is aimed to implement #91, however, in practice, the server closes the current session instead of the old one. I talked with Timo about the issue, and I have to keep the feature commented until it gets fixed or we figure out what is the actual problem.